src/agent/at-driver.js

```diff
@@ -66,20 +73,84 @@ export class ATDriver {
    */
   async *speeches() {
     for await (const message of this._messages()) {
-      if (message.type === 'event' && message.name === 'speech') {
-        yield message.data;
+      if (message.method === 'interaction.capturedOutput') {
+        yield message.params.data;
       }
     }
   }
 }
 
+const seleniumKeysMap = {
```

Comment:
Want to leave a comment that the source for this mapping was https://github.com/SeleniumHQ/selenium/blob/63e8543a39778a5fa65ff2149597066abdbb9abb/py/selenium/webdriver/common/keys.py